WBooks/TableBooks/CellBook/CellBookViewModel.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  CellBookViewModel.swift
+//  WBooks
+//
+//  Created by Juan Silva on 11/06/2021.
+//
+
+import Foundation
+
+final class CellBookViewModel {
+    private var book: Book
```

Comment:
The property will not change, so we could declare it as `let` rather than `var`.